File: stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/ChatMarkdown.java
package com.getstream.sdk.chat;

import android.widget.TextView;

import androidx.annotation.NonNull;

import kotlin.Deprecated;

@Deprecated(message = "ChatMarkdown is deprecated in favour of ChatMessageTextTransformer. " +
        "Use ChatMessageTextTransformer for text ui transformations."
)
public interface ChatMarkdown {
    void setText(@NonNull TextView textView, @NonNull String text);
}

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/markdown/ChatMarkdown.kt
package io.getstream.chat.android.ui.common.markdown

import android.widget.TextView
import kotlin.Deprecated

@Deprecated(
    message = "ChatMarkdown is deprecated in favour of ChatMessageTextTransformer. " +
        "Use ChatMessageTextTransformer for text ui transformations."
)
public fun interface ChatMarkdown {
    public fun setText(textView: TextView, text: String)
}
